src/main/java/org/graylog/integrations/aws/inputs/AWSInput.java

```diff
@@ -145,6 +150,34 @@ public ConfigurationRequest combinedRequestedConfiguration() {
                     "Role ARN with required permissions (cross account access)",
                     ConfigurationField.Optional.OPTIONAL));
 
+            request.addField(new TextField(
+                    CK_CLOUDWATCH_ENDPOINT,
+                    "AWS CloudWatch Override Endpoint",
+                    "",
+                    AWS_SDK_ENDPOINT_DESCRIPTION,
+                    ConfigurationField.Optional.OPTIONAL));
+
+            request.addField(new TextField(
+                    CK_DYNAMODB_ENDPOINT,
+                    "AWS DynamoDB Override Endpoint",
+                    "",
+                    AWS_SDK_ENDPOINT_DESCRIPTION,
+                    ConfigurationField.Optional.OPTIONAL));
+
+            request.addField(new TextField(
```

Comment:
@claudia - Do we want to keep this endpoint or remove IAM?

Reply:
We should keep it in my opinion. I have added more details in this comment:
https://github.com/Graylog2/graylog-plugin-integrations/pull/279#issuecomment-566003682